app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -690,9 +701,65 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
     }
 
     fun animateTabsButton() {
+        toolbarHideHandler.ensureDisplayed()
         binding.pageToolbarButtonTabs.updateTabCount(true)
     }
 
+    // TODO: remove when ABC test is complete.
+    private fun setupNotificationsTest() {
+        binding.pageToolbarButtonNotifications.isVisible = false
+        binding.unreadDotView.isVisible = false
+        when (notificationsABCTestFunnel.aBTestGroup) {
+            0 -> {
+                binding.pageToolbarButtonNotifications.setIcon(R.drawable.ic_inbox_24)
+            }
+            1 -> {
+                binding.pageToolbarButtonNotifications.setIcon(R.drawable.ic_notifications_black_24dp)
+            }
```

Comment:
Minor but the parentheses can be removed.

Reply:
ok!